### PR TITLE
Port prepaid card parts

### PR DESCRIPTION
**Pull request by Aierie**
CS-830 
Styled prepaid card customization-related components.  

Remaining:
- [x] Add prepaid card states to control typography
- [x] Fit themes in
- [x] 'Transparent'/null theme and background handling
- [x] Validation
- [x] Storage of selected customizations in workflow session
- [x] Tests

### CODE REVIEW

**Comment by Aierie** [comment]
The themes are 'mock themes' for now, since XD is too low res to do much. Will ask design for the real themes and add on later.

File: packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.hbs [hunk 1]
```diff
@@ -2,17 +2,58 @@
...
+      @themeOptions={{this.themeOptions}}
+      @headerBackground={{this.headerBackground}}
+      @updateHeaderBackground={{this.updateHeaderBackground}}
+      @headerTheme={{this.headerTheme}}
+      @updateHeaderTheme={{this.updateHeaderTheme}}
+    />
+  {{/if}}
+
+  <div class="layout-customization__disclaimer">
+    * The issuer name will be published on the blockchain and cannot be deleted. Please don’t include any personally identifiable information that you want to keep private.
```
Comment (lukemelia) [inline packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.hbs:30]: I don't think we're going to publish the issuer name to the blockchain.

Maybe better to say: " * The issuer name will be visible to others. Please don’t include any personally identifiable information that you want to keep private."

File: packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    '#37eb77',
+    '#ac00ff',
+    '#efefef',
+    '#393642',
+    '#c3fc33',
+    'linear-gradient(49.27deg, #ff5050 0%, #ac00ff 100%)',
+    'linear-gradient(49.27deg, #03c4bf 0%, #3689d2 30%, #ac00ff 100%)',
+    'linear-gradient(49.27deg, #c3fc33 0%, #0069f9 100%)',
+    'linear-gradient(49.27deg, #ac00ff 0%, #ffd800 100%)',
+    'transparent',
```
Comment (lukemelia) [inline packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts:33]: We're going to want this to come from a service (and then from our API) and for each option to have an ID associated with it. No need to change code for this PR, just FYI.

File: packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  @action save() {
+    if (this.issuerName && this.headerBackground && this.headerTheme) {
+      this.args.workflowSession.update('issuerName', this.issuerName);
+      this.args.workflowSession.update('headerTheme', this.headerTheme);
+      this.args.workflowSession.update(
+        'headerBackground',
+        this.headerBackground
+      );
```
Comment (lukemelia) [inline packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts:82]: Perhaps we should introduce an API that accepts a POJO to set multiple properties at once?

File: packages/web-client/tests/acceptance/issue-prepaid-card-test.ts [hunk 4]
```diff
@@ -90,12 +91,28 @@ module('Acceptance | issue prepaid card', function (hooks) {
...
     post = postableSel(1, 1);
 
-    // // layout-customization card
-    // TODO verify and interact with layout customization card default state
+    assert.dom('[data-test-layout-customization-form]').isVisible();
+    assert
+      .dom(`${post} [data-test-boxel-action-chin] [data-test-boxel-button]`)
+      .isDisabled();
+
+    await fillIn('[data-test-layout-customization-name-input]', 'JJ');
```
Comment (lukemelia) [inline packages/web-client/tests/acceptance/issue-prepaid-card-test.ts:99]: Should we also be selecting some background option(s) in this test to cover the functionality you've added?

**Review by lukemelia — CHANGES_REQUESTED** [review]

File: packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.hbs (hunk 1 above)
Comment (Aierie) [reply packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.hbs:30]: How are we going to store the issuer name prior to persistence?

File: packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts (hunk 3 above)
Comment (Aierie) [reply packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/layout-customization/index.ts:82]: This seems like a good change! Esp if some properties must be updated simultaneously for logic. Will do this.

**Comment by Aierie** [comment]
Keeping small commits for now, will rebase prior to merge

**Review by burieberry — APPROVED** [review]
Looks good! Either in this PR or in a following one, make sure there's a dot in "DAI.CPXD" and that some of the capitalized stuff that Katrin mentioned is lowercased. There's a lot of newly added hardcoded text so not all my changes seem to have made it through.

One thing I noticed is in the XD for prepaid cards, when the background color is darker colored, the header text is white.

**Pull request merged**